boards/DCM/Inc/App/App_SetPeriodicCanSignals.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "App_DcmWorld.h"
+
+struct DcmWorld;
+
+/**
+ * Convert pedal percentage into CAN torque request
+ * @param world DcmWorld needed for canRx and canTx interface
+ */
```

Comment:
```suggestion
```

Reply:
nit; since setting CAN signals is pretty straightforward we can leave the javadoc out